mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -18,15 +28,24 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_8202 getRiptide (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
 	METHOD method_8203 getEquipmentLevel (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;)I
-		ARG 0 ench
+		COMMENT Returns the highest level of the passed enchantment in the enchantment's
+		COMMENT applicable equipment slots' item stacks.
+		ARG 0 enchantment
 		ARG 1 entity
 	METHOD method_8204 getRandomEnchantedEquipment (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;)Ljava/util/Map$Entry;
```

Comment:
Similarly, this could be renamed to `getEquipmentEnchantedWith`

Reply:
renamed to `chooseEquipmentWith`, so you call it like `EnchantmentHelper.chooseEquipmentWith(Enchantments.MENDING, entity)`.